packages/mds-metrics/types.ts

```diff
@@ -76,7 +76,7 @@ export type GetStateSnapshotResponse = MetricsApiResponse<StateSnapshotResponse[
 export type GetEventsSnapshotResponse = MetricsApiResponse<EventSnapshotResponse[]>
 export type GetTelemetryCountsResponse = MetricsApiResponse<TelemetryCountsResponse[]>
 export type GetEventCountsResponse = MetricsApiResponse<EventCountsResponse[]>
-export type GetAllResponse = MetricsApiResponse<MetricsAllResponse[]>
+export type GetAllResponse = MetricsApiResponse<MetricsAllResponse[] | string>
```

Comment:
This is curious. Is this temporary?  Perhaps a separate type for getting `tsv` formatted response?

Reply:
I still need to work with Owen to finalize the API response shape. This is temporary-ish, but may end up getting finalized if Owen decides he doesn't want to ship the TSV with "bucketing".

Reply:
Added a comment to explain payload shape